guides/data.json

```diff
@@ -1,4 +1,9 @@
 [
+	{"source": "page-visibility-api"},
```

Comment:
Did you add these new posts?

Reply:
No, they are duplicate sources. I will remove them.